sintia/sintia.py

```diff
@@ -835,6 +867,7 @@ async def on_message(self, message: discord.Message) -> None:
         await asyncio.gather(
             self.vote_handler(message),
             user_stats.record_message(message),
+            self.chat_message_handler(message),
```

Comment:
```suggestion
            self.chat_message_handler(self.irc_bridge, message),
```

Reply:
Not needed, `self` already has `irc_bridge`